gradle-baseline-java/src/main/groovy/com/palantir/baseline/extensions/BaselineErrorProneExtension.java

```diff
@@ -25,6 +25,7 @@
             // Baseline checks
             "AssertjPrimitiveComparison",
             "CatchBlockLogException",
+            "ExceptionSpecificity",
```

Comment:
gotta break this out into the new CatchSpecificity and ThrowsSpecificity

Reply:
Doh thanks!